api/params.go

```diff
@@ -74,7 +73,7 @@ func (s *ParamString) UnmarshalText(b []byte) error {
 }
 
 // String implements fmt.Stringer.
-func (t TimeRFC3339) String() string { return url.QueryEscape((time.Time)(t).Format(time.RFC3339)) }
+func (t TimeRFC3339) String() string { return (time.Time)(t).Format(time.RFC3339Nano) }
```

Comment:
just to make absolutely sure: we're keeping the `time.RFC3339` when marshalling to JSON?

Reply:
Yeah we just don't URL escape here anymore since `url.Values` already does that with every string.